backend/hardware/reagentdispenser/syringepump.py

```diff
@@ -58,13 +57,12 @@ def __init__(self, reagent_dispenser_config: dict):
             }
             self.axisMinmmPerMin[axis] = math.ceil((30 / stepsPerMM) * 120)
             # configure steps/mm
-            self.grblWrite(
-                self.grblSer, "$10{0}={1}\n".format(axisToCNCID[axis], stepsPerMM)
+            self.grbl.grblWrite(
+                "$10{0}={1}".format(axisToCNCID[axis], stepsPerMM)
             )
             # configure max mm/min
-            self.grblWrite(
-                self.grblSer,
-                "$11{0}={1}\n".format(axisToCNCID[axis], syringeConfig["maxmmPerMin"]),
+            self.grbl.grblWrite(
+                "$11{0}={1}".format(axisToCNCID[axis], syringeConfig["maxmmPerMin"]),
```

Comment:
Those settings are saved to grbl eeprom. This may cause a problem when user switches from syringe pump back to peristaltic pump and old settings may cause the pump to use wrong step count and max speed.
I suppose peristaltic pump may need to have those settings be enforced too.

Also $110/111/112 max rate may be redundant because `G1 (...) F{value}` already sets max speed for the motor for single operation.

Reply:
Good catch, I made a new issue for this to fix in another PR.